### PR TITLE
feat(union-page): create what is a debtors union section

### PR DESCRIPTION
**Pull request by arathjz**
**What:**
Create "what is a debtors union" section

**Why:**
Relates to: https://app.asana.com/0/1159164196409129/1191532016000025/f

**How:**
- Include section into the debtors union page

#### Media
https://www.loom.com/share/bed6505dc7fa46b9bb061be4deb90770

### CODE REVIEW

File: src/sections/WhatIsADebtorsUnion/index.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            interests against greedy bosses and owners. Similarly, debtors are
+            coming together to form a union to defend our interests. We use our
+            collective power to fight for better conditions in our financial
+            lives and a more equitable society: the cancellation and
+            renegotiation of debts, access to publicly-funded goods (education,
+            healthcare, housing, etc), anti-racist economic policies, and much
+            more.
+          </p>
+        </div>
+        <Image
```
Comment (duranmla) [inline src/sections/WhatIsADebtorsUnion/index.tsx:52]: I guess we are gonna do something in regards different assets for different devices later on... Do we have a ticket for that?

**Review by duranmla — APPROVED** [review]
Remember to give a quick look on other browsers. Mostly FF.
LGTM!

File: src/sections/WhatIsADebtorsUnion/index.tsx (hunk 1 above)
Comment (arathjz) [reply src/sections/WhatIsADebtorsUnion/index.tsx:52]: Yes we do https://app.asana.com/0/1159164196409129/1192037358959244/f

**Pull request merged**